Doxyfile

```diff
@@ -1319,7 +1319,7 @@ CHM_FILE               =
 HHC_LOCATION           =
 
 # The GENERATE_CHI flag controls if a separate .chi index file is generated
-# (YES) or that it should be included in the master .chm file (NO).
+# (YES) or that it should be included in the primary .chm file (NO).
```

Comment:
These comments are autogenerated by doxygen. If the upstream has changed what they generate here, I am fine with an update, but otherwise I don't want to add unnecessary diff here.

Reply:
Indeed, if you generate a template file with latest version of doxygen, it generates here a reference with "in the main .chm file (NO)". I will change it